CMakeLists.txt

```diff
@@ -100,6 +100,23 @@ if (UNIX)
     install(DIRECTORY include/ DESTINATION /usr/local/include/uvgrtp
             FILES_MATCHING PATTERN "*.hh"
     )
+
+    # Find pkg-config installed in the system
+    find_package(PkgConfig REQUIRED)
+    if(PkgConfig_FOUND)
+        # Check PKG_CONFIG_PATH, if not defined, use /usr/local/lib/pkgconfig
+        if(NOT DEFINED ENV{PKG_CONFIG_PATH})
+            set( PKG_CONFIG_PATH "/usr/local/lib/pkgconfig")
+            message("PKG_CONFIG_PATH is not found. Set it with ${PKG_CONFIG_PATH}")
```

Comment:
PKG_CONFIG_PATH is not set. Setting it to ${PKG_CONFIG_PATH}.

I think this gives a better impression of what is happening.